README.md

```diff
@@ -1,27 +1,43 @@
-# Project Name
+# Magazine
 
-> One paragraph statement about the project.
+> A magazine article using CSS Grid (grid rows and grid columns)
```

Comment:
Take a look at the project paragraph statement. I think it should be A magazine article built with the use of grid(grid rows and grid columns) and media queries.